demo/prediction/js/prediction-use-case.js

```diff
@@ -16,20 +16,20 @@ class PredicatedLateUseCase extends UseCase {
         this._initManagers();
 
         this._style.reduceVisibilityOfExecutedElements(this._executionData.executedElements);
-        this._style.highlightRunningElementsWithPrediction(this._executionData.runningElementWithPrediction);
+        this._style.highlightRunningElementWithPrediction(this._executionData.runningElementWithPrediction);
         this._style.toggleHighlightRunningElementsWithoutPrediction(this._executionData.runningElementsWithoutPrediction);
-        this._registerInteractions(this._executionData.predictedPaths, this._executionData.runningElementsWithoutPrediction, this._executionData.runningElementWithPrediction);
+        this._registerInteractions(this._executionData.runningElementWithPrediction.id, this._executionData.predictedPaths, this._executionData.runningElementsWithoutPrediction);
     }
 
     _initManagers() {
-        this._style = new PredicatedLateStyle(this._bpmnVisualization.bpmnElementsRegistry);
+        this._style = new PredicatedLateStyle(this._bpmnVisualization.bpmnElementsRegistry, `${this.type}-bpmn-container`);
```

Comment:
**question:** can't we retrieve the id from the UseCase class? we are leaking the internal implementation of UseCase here!